.travis/travisci-deploy-key.enc

```diff
@@ -0,0 +1 @@
+{your.encrypted.private.key}
```

Comment:
` .travis/travisci-deploy-key.enc` is from this command `travis encrypt-file --repo pathikrit/better-files {your.private.key} {better-files.root.folder}/.travis/travisci-deploy-key.enc`